src/modules/developer-guide/pages/server-plugins/index.adoc

```diff
@@ -0,0 +1,145 @@
+include::ROOT:partial$variables.adoc[]
+
+= Server plugins
+
+{PRODUCT_NAME} server can be extended using https://pkg.go.dev/plugin[Go plugins].
+You can modify default behaviour of the server and provide new functionalities like workflow functions and types.
+
+== Basic plugin structure
```

Comment:
Please add an outline of the FS; you can use http://localhost:8080/corteza-docs/2021.9/integrator-guide/automation/automation-scripts/index.html#_file_structure as an example.

This tells me how a plugin looks like, but it doesn't tell me how to structure it.

Reply:
Agreed. But let's wait a bit with this part -- we have to figure this one out on our side a s well before trying to teach others.